decode/render.ts

```diff
@@ -8,27 +8,18 @@ let boxmodels = {};
 let metrics: MetricData = null;
 let lean = false;
 const METRIC_MAP = {};
-METRIC_MAP[Metric.Nodes] = { name: "Node Count", unit: ""};
+METRIC_MAP[Metric.Bytes] = { name: "Bytes", unit: "KB"};
 METRIC_MAP[Metric.LayoutBytes] = { name: "Layout Bytes", unit: "KB"};
 METRIC_MAP[Metric.InteractionBytes] = { name: "Interaction Bytes", unit: "KB"};
 METRIC_MAP[Metric.NetworkBytes] = { name: "Network Bytes", unit: "KB"};
-METRIC_MAP[Metric.DiagnosticBytes] = { name: "Diagnostic Bytes", unit: "KB"};
-METRIC_MAP[Metric.Mutations] = { name: "Mutation Count", unit: ""};
-METRIC_MAP[Metric.Interactions] = { name: "Interaction Count", unit: ""};
```

Comment:
>Interactions [](start = 18, length = 12)

Worth keeping? Your call.

Reply:
Interactions was also a count of events like: Scroll, Pointer, Selection, etc. I think based on our offline thread, you could easily compute it with the events we have. It essentially is a count of events.